src/Sylius/Bundle/CoreBundle/Form/Type/CurrencyType.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\Type;
+
+use Sylius\Bundle\CurrencyBundle\Form\Type\CurrencyType as BaseCurrencyType;
+use Sylius\Component\Currency\Model\CurrencyInterface;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+
+/**
+ * @author Jan Góralski <jan.goralski@lakion.com>
+ */
+class CurrencyType extends BaseCurrencyType
```

Comment:
Btw. I guess we can achieve the same result by using [form type extensions](http://symfony.com/doc/2.7/form/create_form_type_extension.html) instead of custom type. It will be also a lot cleaner.